src/stores/solution-create.js

```diff
@@ -1,18 +1,12 @@
 import { defineStore } from 'pinia'
 
-export const useSolutionCreate = defineStore({
+export const useSolutionStore = defineStore({
   id: 'solutionCreate',
   state: () => ({
     solution: null
   }),
-  persist: true,
-  getters: {
-    getSolution() {
-      return this.solution
-    }
-  },
   actions: {
-    addSolution(solution) {
+    setSolution(solution) {
       this.solution = solution
```

Comment:
Could you create a mutation for this? The mutation changes the state and the action calls the mutation. So this way the action doesn't change the state directly